### PR TITLE
Add MariaDB databases

### LINKED ISSUE #169: Add MariaDB database
`backend/3-auth` and `backend/4-e3selector` services requires an additional SQL database.

### PR DESCRIPTION
**Pull request by shoebjoarder**
Probably we need to install the python3 and MySQL development headers and libraries using: `sudo apt-get install python3-dev default-libmysqlclient-dev build-essential`

### CODE REVIEW

**Review by ralf-berger — CHANGES_REQUESTED** [review]
Not ready:
* does not build
* Kubernetes config missing

**Comment by shoebjoarder** [comment]
Ah, didn't notice that you created a branch!

File: backend/3-auth/Pipfile.lock
Comment (ralf-berger) [inline backend/3-auth/Pipfile.lock:1]: Breaks build for me, `jellyfish-1.0.1` fails with `Checking for Rust toolchain....`

File: backend/4-e3selector/Pipfile.lock
Comment (ralf-berger) [inline backend/4-e3selector/Pipfile.lock:1]: Breaks build for me, `jellyfish-1.0.1` fails with `Checking for Rust toolchain....`

Comment (shoebjoarder) [reply backend/4-e3selector/Pipfile.lock:1]: For me the build was successful when `default-libmysqlclient-dev` added to Dockerfile.

```Dockerfile
ENV BUILD_DEPS "build-essential pkg-config libpq-dev default-libmysqlclient-dev"
```

File: backend/3-auth/Pipfile.lock
Comment (shoebjoarder) [reply backend/3-auth/Pipfile.lock:1]: For me the build was successful when `default-libmysqlclient-dev` added to Dockerfile.

```Dockerfile
ENV BUILD_DEPS "build-essential pkg-config libpq-dev default-libmysqlclient-dev"
```

File: backend/4-e3selector/Pipfile.lock
Comment (shoebjoarder) [reply backend/4-e3selector/Pipfile.lock:1]: Same, added `default-libmysqlclient-dev` to successfully build it.

```Dockerfile
ENV BUILD_DEPS "build-essential pkg-config libpq-dev default-libmysqlclient-dev"
```

File: backend/3-auth/Pipfile.lock
Comment (ralf-berger) [reply backend/3-auth/Pipfile.lock:1]: x86 emulation seems to works

**Pull request closed without merging**